feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -99,34 +114,42 @@ fun ProfileView(
                     bottom = sizes.smallerSize
                 )
         ) {
-            Icon(
-                Icons.Outlined.Phone,
-                contentDescription = stringResource(id = R.string.phone_icon),
-                modifier = Modifier.padding(end = sizes.minorSmallSize),
-                tint = MaterialTheme.colorScheme.tertiary
-            )
-            Text(
-                text = "(404) 979-2400",
-                fontWeight = FontWeight.Normal,
-                color = MaterialTheme.colorScheme.tertiary
-            )
+            AnimatedVisibility(true) {
+                Icon(
+                    Icons.Outlined.Phone,
+                    contentDescription = stringResource(id = R.string.phone_icon),
+                    modifier = Modifier.padding(end = sizes.minorSmallSize),
+                    tint = MaterialTheme.colorScheme.tertiary
+                )
+            }
+            AnimatedVisibility(true) {
```

Comment:
If you are going to add an animation, please just add the name of the animation in the name arguments.

Reply:
Working on